src/wp-admin/includes/schema.php

```diff
@@ -580,6 +580,9 @@ function populate_options( array $options = array() ) {
 
 		// 6.4.0
 		'wp_attachment_pages_enabled'     => 0,
+
+		// CP-2
```

Comment:
Can this be extended to CP-2.3.0and we can get it into core for the next minor release.

Reply:
Wondering if there will be a 2.2.1.

Reply:
Next version is probably for discussion at the next core meeting, leave this for now until we can discuss and decide.

Reply:
Moved to CP-2.3.0.